docker-compose.yml

```diff
@@ -1,5 +1,6 @@
 volontulo:
-  build: .
+  image: codeforpoznan/volontulo
+  # build: .
```

Comment:
I leave it on purpose. Somethimes You want to build image instead of pulling from dockerhub, so instead of typing `build: .` You can just uncomment it here and comment `image:` line.